packages/vscode-extension/src/extension.ts

```diff
@@ -35,6 +35,21 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(new ExtTelemetry.Reporter(context));
 
   await exp.initialize(context);
+  TreatmentVariableValue.isExpandCard = (await exp
```

Comment:
This logic has been removed in dev since the experiment is done

Reply:
got it.